ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml

```diff
@@ -209,14 +201,6 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-automation-flavors-aks-e2e
   timeout: 5h0m0s
-- as: aks-ebpf-qa-e2e-tests
-  postsubmit: true
-  steps:
-    env:
-      COLLECTION_METHOD: ebpf
-      TEST_SUITE: aks-qa-e2e-tests
```

Comment:
The non-ebpf version does not have TEST_SUITE: eks-qa-e2e-tests. Perhaps that should be added.

Reply:
Replied in <https://github.com/openshift/release/pull/64474#discussion_r2073247359>. Here it is similar.